core/src/impl/Kokkos_Error.hpp

```diff
@@ -66,6 +76,17 @@ void traceback_callstack( std::ostream & );
 
 std::string human_memory_size(size_t arg_bytes);
```

Comment:
https://en.wikipedia.org/wiki/The_Magical_Number_Seven,_Plus_or_Minus_Two

Reply:
I honestly don't know what this function does :)